### PR TITLE
Upgrade decidim 0.15.2

### PR DESCRIPTION
**Pull request by aitorlb**
- Update `decidim` version `'0.15.2'`
- Update tags:
  - `gem 'decidim-cas_client', tag: 'v0.0.17'`
- Run migrations
- Add upgrade notes
- Add whenever gem

**Pending:** testing `decidim-cas_client` with new decidim version

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -11,6 +11,8 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
```
Comment (tramuntanal) [inline config/application.rb]: we're on Rails 5.2 now, shouldn't we update this sentence?

File: docs/UPGRADE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Update your Gemfile:
+```
+gem "decidim", "0.XX.X"
+gem "decidim-dev", "0.XX.X"
+```
+
+Run these commands to upgrade and make sure you get all the latest migrations:
+```
+bundle update decidim
+bin/rails decidim:upgrade
```
Comment (tramuntanal) [inline docs/UPGRADE.md:15]: The same I recommended for participa.somenergia.coop here, better explain what should be executed locally and what in the server.

**Review by tramuntanal — CHANGES_REQUESTED** [review]
Bé!

File: docs/UPGRADE.md (hunk 2 above)
Comment (aitorlb) [reply docs/UPGRADE.md:15]: Thanks

File: config/application.rb (hunk 1 above)
Comment (aitorlb) [reply config/application.rb]: Thanks

**Review by tramuntanal — APPROVED** [review]
yeah!

**Pull request merged**